src/services/io/podio/JEventProcessorPODIO.cc

```diff
@@ -394,6 +394,13 @@ JEventProcessorPODIO::JEventProcessorPODIO() {
       "EcalEndcapNTrackClusterMatches",
       "HcalEndcapNTrackClusterMatches",
 
+      // particle flow
+      /* TODO move PFA0 collections here when ready */
+      /* TODO add PFA1 collections here */
+      /* TODO add PFA2 collections here */
+      "ReconstructedNeutralCandidates",
+      /* TODO add PFA3 collections here */
```

Comment:
Propagating the above comments
```suggestion
      "EndcapNNeutralCandidateParticlesAlpha",
```